kin-sdk/kin-sdk-lib/src/androidTest/kotlin/kin/sdk/KinAccountIntegrationTest.kt

```diff
@@ -110,23 +111,59 @@ class KinAccountIntegrationTest {
         val memo = "fake memo"
         val expectedMemo = addAppIdToMemo(memo)
 
-        val latch = CountDownLatch(1)
-        val listenerRegistration = kinAccountReceiver.addPaymentListener { _ -> latch.countDown() }
+        val transactionId = sendTransactionAndAssert(kinAccountSender, kinAccountReceiver, memo)
 
-        val transaction = kinAccountSender.buildTransactionSync(kinAccountReceiver.publicAddress.orEmpty(),
-                BigDecimal("21.123"), fee, memo)
-        val transactionId = kinAccountSender.sendTransactionSync(transaction)
-        assertThat(kinAccountSender.balanceSync.value(), equalTo(BigDecimal("78.87700").subtract(feeInKin)))
-        assertThat(kinAccountReceiver.balanceSync.value(), equalTo(BigDecimal("21.12300")))
+        val server = Server(TEST_NETWORK_URL)
+        val transactionResponse = server.transactions().transaction(transactionId.id())
+        val actualMemo = transactionResponse.memo
+        assertTrue { actualMemo is MemoText }
+        assertThat((actualMemo as MemoText).text, equalTo(expectedMemo))
```

Comment:
you can emit the previous assertTrue because it is actually included in this one